docs/api/wanchain.md

```diff
@@ -2091,15 +2138,15 @@ This order kind is used for delegating `ValueLedger` deploy.
 |-|-
 | expiration | [required] An `integer` number representing the timestamp in milliseconds after which the order expires and can not be performed any more.
 | kind | [required] An `integer` number that equals to `OrderKind.ASSET_LEDGER_DEPLOY_ORDER`.
-| makerId | [required] A `string` representing an Ethereum account address which makes the order. It defaults to the `accountId` of a provider.
+| makerId | [required] A `string` representing an Wanchain account address which makes the order. It defaults to the `accountId` of a provider.
```

Comment:
| makerId | [required] A `string` representing a Wanchain account address which makes the order. It defaults to the `accountId` of a provider.